app/src/main/res/values-sv/strings.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE resources [
+	<!ENTITY appname "Slideshow">
+	<!ENTITY author "Michael Standen">
+	]>
+
+<resources>
+	<string name="app_name">&appname;</string>
+	<string name="action_settings">Inställningar</string>
+	<string name="action_credits">Om appen</string>
+
+	<!-- List -->
+	<string name="inaccessible">(otillgänglig)</string>
+	<string name="go_home_folder">Fastnat? Klicka för att komma till hemskärmen</string>
+	<string name="play_folder">Spela upp bildspel härifrån</string>
+
+	<!-- Slideshow -->
+	<string name="title_activity_image">ImageActivity</string>
+
+	<string name="delete_button">Ta bort</string>
+	<string name="delete_dialog_title">Ta bort bild</string>
+	<string name="delete_dialog_message">Är du <s>				</s>äker?</string>
```

Comment:
Can you explain what the `<s> </s>` is intended for? 
I can't be sure if this is a language typical thing or just an error